syntax = "proto2";

package st;

import "modules/cnoa_pnc/planning/proto/affine_transformation.proto";

message PiecewiseLinearFunctionDoubleProto {
  repeated double x = 1;
  repeated double y = 2;
}

message PiecewiseLinearFunctionVec2dProto {
  repeated double x = 1;
  repeated Vec2dProto y = 2;
}

message PiecewiseLinearFunctionVec3dProto {
  repeated double x = 1;
  repeated Vec3dProto y = 2;
}

message PiecewiseLinearFunctionVec4dProto {
  repeated double x = 1;
  repeated Vec4dProto y = 2;
}